apps/styleguide/src/mixins/TableMetadataMixin.vue

```diff
@@ -22,7 +22,7 @@ export default {
       this.graphqlError = null;
       request(
         this.graphqlURL,
-        "{_session{email,roles}_schema{name,tables{name,description,externalSchema,semantics,columns{name,columnType,key,refTable,refLink,refLabel,refBack,required,semantics,description}settings{key,value}}}}"
+        "{_session{email,roles}_schema{name,tables{name,description,externalSchema,semantics,columns{name,columnType,key,refTable,refLink,refLabel,refBack,required,semantics,description,position}settings{key,value}}}}"
```

Comment:
added position, allows sorting in the UI

Reply:
these 500 + char string are getting hard to maintain , suggest to format using gql

Reply:
would be great if we have a linter who can do this for us?